#! /bin/sh /usr/share/dpatch/dpatch-run
## 11_mtd_logging.dpatch by  <laga@laga.ath.cx>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix logging location for mtd. On upgraded installs, you need to enable
## DP: logging in MythVideo settings

@DPATCH@
diff -urNad mythplugins~/mythvideo/mtd/logging.cpp mythplugins/mythvideo/mtd/logging.cpp
--- mythplugins~/mythvideo/mtd/logging.cpp	2009-03-22 17:15:54.000000000 -0500
+++ mythplugins/mythvideo/mtd/logging.cpp	2009-03-22 17:19:49.000000000 -0500
@@ -27,15 +27,7 @@
 
 bool MTDLogger::Init(void)
 {
-    QString logfile_name = gContext->GetSetting("DVDRipLocation");
-    if (logfile_name.isEmpty())
-    {
-        VERBOSE(VB_IMPORTANT, LOC_ERR +
-                "You do not have a DVD rip directory set. Run Setup.");
-        return false;
-    }
-
-    logfile_name += "/mtd.log";
+    QString logfile_name = QString("/var/log/mythtv/mtd.log");
 
     if (!log_to_stdout)
     {
diff -urNad mythplugins~/mythvideo/mythvideo/globalsettings.cpp mythplugins/mythvideo/mythvideo/globalsettings.cpp
--- mythplugins~/mythvideo/mythvideo/globalsettings.cpp	2009-03-22 17:15:54.000000000 -0500
+++ mythplugins/mythvideo/mythvideo/globalsettings.cpp	2009-03-22 17:19:49.000000000 -0500
@@ -510,8 +510,7 @@
     gc->setValue(false);
     gc->setHelpText(QObject::tr("If set, the MTD (Myth Transcoding Daemon) "
                     "will log to the window it is started from. "
-                    "Otherwise, it will write to a file called  "
-                    "mtd.log in the top level ripping directory."));
+                    "Otherwise, it will write to /var/log/mythtv/mtd.log"));
     return gc;
 }
 
